js/custom.js

```diff
@@ -23,4 +23,35 @@ $(document).ready(function () {
 });
 $('.map-holder').click(function () {
     $(this).removeClass('zoom-off');
-});
\ No newline at end of file
+});
+
+function readMore(number) {
+  if (number == 1) {
+    var dots = document.getElementById("dots1");
+    var moreText = document.getElementById("more1");
+    var btnText = document.getElementById("myBtn1");
+
+  } else if (number == 2) {
+    var dots = document.getElementById("dots2");
+    var moreText = document.getElementById("more2");
+    var btnText = document.getElementById("myBtn2");
+
+  } else if (number == 3) {
+    var dots = document.getElementById("dots3");
+    var moreText = document.getElementById("more3");
```

Comment:
Append the number to the IDs and reduce the ifs.

Reply:
I tried that, I failed terribly.

Reply:
It won't fail, please do it correctly.